repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -156,6 +160,10 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:assert vc:minVersion="1.1"
+                           test="count((@via, ./papic:via-header)) le 1"
```

Comment:
I was just following the existing, but since we're gutting things we might as well do them right.